doc/versioning_and_life_cycle.md

```diff
@@ -0,0 +1,73 @@
+
+- For the private preview api-version, we usually do not recommend releasing those private features in the core CLI/CLI extension, so as to avoid premature exposure of private features. It is suggested to use [edge build](https://github.com/Azure/azure-cli/blob/dev/doc/try_new_features_before_release.md) to build the private packages to meet the urgent needs of some customers first.
+
+Since the release states of api-version and CLI commands may be inconsistent, our intent is for customers to only have to pay attention to the release states *(`is_preview` flag)* of the command itself, rather than the api-version used behind it. This will be easier for customers to understand.
+
+## Breaking Changes
+
+### What is a breaking change
+
+A breaking change is any update to Azure CLI, including core CLI and CLI extensions, that causes a customer's script or automation, written in a previous version, to fail.
+Common examples of breaking changes include:
+- Modifying names of parameters/commands.
+- Modifying input logic of parameters.
+- Modifying the format or properties of result output.
+- Modifying the current behavior model.
+- Adding additional verification that changes CLI behavior.
+- Removing parameters/commands
+
+All breaking changes for commands will be marked as **\[BREAKING CHANGE\]** in [release notes](https://docs.microsoft.com/cli/azure/release-notes-azure-cli).
+
+### When do breaking changes occur
+
+Azure CLI has bi-annual breaking change releases coinciding with Microsoft **Build** and **Ignite** events. Azure CLI **generally** releases the breaking changes **only** in these two breaking change windows. This approach allows customers to maintain stability, especially in automation scenarios, while keeping up-to-date with the latest and most secure versions of Azure CLI. It also allows customers to plan, test, and migrate Azure CLI commands and features properly when breaking changes are released.
+
+However, there are special cases where breaking changes need to be released as soon as possible. In the following cases, a breaking change to a command or feature **doesn't** need to wait for the next breaking change release cycle and can be released as soon as the change is ready.
+
+- A critical bug is affecting customers and requires a hot fix.
+- A security patch is required. 
```

Comment:
```suggestion
- A security or data loss related fix is required. 
```

Reply:
I'd like to ask what the `data` in `data loss related fix` refers to? Is it the loss of customer's business data or the loss of the Telemetry data we collected or the loss of customer traffic?

Reply:
customer loosing their data due to the client tools.

Reply:
Got it, thanks~